### PR TITLE
Fix/doc

### PR DESCRIPTION
**Pull request by sgmac**
I changed some names in the doc/YAML to make it work.

### CODE REVIEW

File: 07-deployment/README.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ Change again the number of replicas to `2`, reapply, see what is happening.
 We know how to scale up/down a deployment, but how can we deploy a new version of the application. To achieve this, we need to tell Kubernetes to update the image we are using in our `deployment`, for this:
 
 ```sh
-$ kubectl set image deployment/simple-deployment simple-service=mhausenblas/simpleservice:0.5.0
+$ kubectl set image deployment/simple-deployment simple-pod=mhausenblas/simpleservice:0.5.0
 deployment.apps "simple-deployment" image updated
```
Comment (sgmac) [inline 07-deployment/README.md]: With version 4.0 I could not access `/info` or `/env`

File: 08-service/01-simple-deployment.yml [hunk 2]
```diff
@@ -15,6 +15,6 @@ spec:
     spec:
       containers:
         - name: simple-pod
-          image: mhausenblas/simpleservice:0.4.0
+          image: mhausenblas/simpleservice:0.5.0
```
Comment (ElPicador) [inline 08-service/01-simple-deployment.yml]: it's on purpose to have `0.4.0` to be able to update it

File: 08-service/03-internal-service.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: simple-service
+  name: simple-internal-service
```
Comment (ElPicador) [inline 08-service/03-internal-service.yml]: why renamaing it?

File: 08-service/04-ingress.yml [hunk 4]
```diff
@@ -7,5 +7,5 @@ metadata:
     nginx.ingress.kubernetes.io/ssl-redirect: "false"
 spec:
   backend:
-    serviceName: simple-service
+    serviceName: simple-internal-service
```
Comment (ElPicador) [inline 08-service/04-ingress.yml]: why renamaing it?

File: 08-service/README.md [hunk 5]
```diff
@@ -86,18 +86,18 @@ is central to Kubernetes. It is with those fields that you will tell Kubernetes
 Apply the service:
 
 ```sh
-$ kubectl apply -f 08-service/03-simple-service.yml
-service "simple-service" created
+$ kubectl apply -f 08-service/03-simple-internal-service.yml
```
Comment (ElPicador) [inline 08-service/README.md]: same

File: 07-deployment/README.md (hunk 1 above)
Comment (ElPicador) [reply 07-deployment/README.md]: Yep it's expected, see: https://github.com/mhausenblas/simpleservice

File: 08-service/03-internal-service.yml (hunk 3 above)
Comment (sgmac) [reply 08-service/03-internal-service.yml]: it wasn't aligned the doc, reference tot `simple-internal-service` but name was the above.

Comment (ElPicador) [reply 08-service/03-internal-service.yml]: 🆗

File: 08-service/01-simple-deployment.yml (hunk 2 above)
Comment (ElPicador) [reply 08-service/01-simple-deployment.yml]: could you remove this change? And I'll merge the PR

File: 07-deployment/README.md (hunk 1 above)
Comment (ElPicador) [reply 07-deployment/README.md]: same for that

**Comment by ElPicador** [comment]
Closing in favor of: https://github.com/algolia/kubernetes-hands-on/pull/36

**Pull request closed without merging**